### ISSUE TITLE
Cant import files after downloading

### ISSUE THREAD

**Issue by Azoz5643** [avluis/Hentoid#381]
**Hentoid Version**: Latest

**Device/Android Version**: 9.0.0

 I downloaded hentoid with my files in my device (I used hentoid few months ago) now when I'm want to import it says importing. After that I have to choose dark mode or light. When I chose any one of them I need to import again, but this time when I tap the app freezes so basically I can't get in the app
**Issue details / Repro steps**:

**Stack trace / LogCat**:
```ruby
paste stack trace and/or log here
```

@AVnetWS/admin-team

**Comment by RobbWatershed** [comment]
Hi and thanks for the report !

Let me rephrase the issue and tell me if I'm correct :
- You have reinstalled Hentoid on a device with some older downloads on it
- When starting the app (v1.9.0) , you see the "Importing..." dialog _twice_ during the startup wizard
- First time (on the "Library" screen), it imports all the books properly (progress bar goes from 0% to 100%)
- Second time (after choosing dark/light mode), the "Importing..." dialog shows up again and the app freezes

=> Can you confirm this is what happens ?
=> Can you tell me if your downloads are on your phone memory or your SD card ?

**Comment by Azoz5643** [comment]
This exactly what happened, but it only freezes when I choose Custom or Default. And the download are on memory cards. Also when I managed to get in but with like 10% of the downloaded stuff. I tried to refresh and the screen turns black plus I checked on .Hentoid all the files are good and have no problems

**Comment by RobbWatershed** [comment]
Have you tried leaving your phone alone while importing, for 10 minutes? 

These screens are bad at handling user input, but the import itself is quite solid. It may just work properly without you knowing...

PS: we plan on improving that soon

**Comment by RobbWatershed** [comment]
Import progress dialog is now impossible to hide by mistake.

That change will be available on next beta.
